cyten/spaces.py

```diff
@@ -2101,6 +2101,9 @@ def take_slice(self, blockmask):
     def with_opposite_duality(self):
         return AbelianLegPipe(legs=self.legs, is_dual=not self.is_dual, combine_cstyle=self.combine_cstyle)
 
+    def with_opposite_duality_and_combinestyle(self):
```

Comment:
This, together with ``Tensor.flip_leg_duality`` seems unrelated?
Should be a separate PR. Do you want to do that or should I?